gcloud-java-storage/src/test/java/com/google/cloud/storage/BucketTest.java

```diff
@@ -249,33 +247,35 @@ public void testGet() throws Exception {
   }
 
   @Test
-  public void testGetAll() throws Exception {
+  public void testGetAllArray() throws Exception {
     initializeExpectedBucket(4);
-    Capture<BatchRequest> capturedBatchRequest = Capture.newInstance();
-    List<Result<Blob>> batchResultList = new LinkedList<>();
-    for (Blob info : blobResults) {
-      batchResultList.add(new Result<>(info));
-    }
-    BatchResponse response = new BatchResponse(Collections.<Result<Boolean>>emptyList(),
-        Collections.<Result<Blob>>emptyList(), batchResultList);
     expect(storage.options()).andReturn(mockOptions);
-    expect(storage.submit(capture(capturedBatchRequest))).andReturn(response);
-    expect(storage.options()).andReturn(mockOptions).times(3);
-    replay(storage);
+    List<BlobId> blobIds = Lists.transform(blobResults, new Function<Blob, BlobId>() {
+      @Override
+      public BlobId apply(Blob blob) {
+        return blob.blobId();
+      }
+    });
+    expect(storage.get(blobIds)).andReturn(blobResults);
+    EasyMock.replay(storage);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'EasyMock.replay' due to existing static import 'org.easymock.EasyMock.replay'](https://www.codacy.com/app/mziccard/gcloud-java/file/2448729656/issues/source?bid=3123103&fileBranchId=3247265#l260)